OVRLighthouseManager/Helpers/LighthouseCommands.cs

```diff
@@ -153,11 +153,12 @@ public class StandbyCommand : ICommand
 
     public bool CanExecute(object? parameter)
     {
-        var isV1 = parameter is LighthouseObject lighthouse && new Lighthouse { Name = lighthouse.Name }.Version == LighthouseVersion.V1;
-        if (isV1)
-        {
-            return false;
-        }
+        // V1s can do standby!

```

Comment:
As far as I know, V1 can't use standby and it's V2 feature. Actually my V1 didn't turn off LEDs, then it gets a state which the status LED alternately blinks green and blue.
Are there any articles or screenshots which show V1 can do standby?